src/main/java/routes/Get.java

```diff
@@ -6,6 +6,7 @@
 public class Get {
     private RequestSpecification request;
     public String route = "/employees";
+    public String route1 = "/employee/1";
```

Comment:
It's not comfortable to use hard-coded values in URL. 
Better to refactor this one
```
public String employeesRoute = "/employees";
public String employeeRoute = "/employee/";
```

After it in method, you can take an additional parameter (ID). And modify String to use this param.
For example (code may be a little bit incorrect)
```
getEmployee(int id) {
    String route = employeeRoute + id;
```